pages/src/source/tutorials/schematics.md

```diff
@@ -341,8 +347,115 @@ You should always apply a [groundlevel tag](../../source/items/tagtool) when mak
+
+|   |   |   |
+|---|---|---|
+|   | X |   |
+| X | P | X |
+|   | X |   |
+
+X = walking position
+
+P = planting position
+
+> *Note*: Kelp is an exception to this behaviour, to prevent them from going diving into the water, the planter will actually walk to the first air block above the water, looking up for 26 blocks (1 above the max plant length), if this is not possible they will not be able to harvest this plant, so ensure there is air present above the water.
+> 
+>  |   |   |   |
+>  |---|---|---|
+>  |   | X |   |
+>  | W | K | W |
+>  | W | K | W |
+>  | W | K | W |
+>  | D | D | D |
+>
+> D = ground
+> 
+> W = water
+> 
+> K = kelp
+> 
+> X = walking position
+
+For downwards growing plants the same rules apply, although they will walk to the top of the stem, so ensure there are some ways for them to get up to the ceiling.
+
+### Treeside plants
+
+Treeside plants speak for themselves, these plants grow on the side of trees. In this case you are supposed to tag the stem of the tree itself, and the working positions will automatically be set to every **horizontally adjacent** block of the tree.
+
+This means that the amount of tags you can actually place is the amount of working positions divided by 4!
+
+Currently this is only used for Cocoa beans.
+
+### Bonemealed fields
+
+Bonemealed fields will tell the planter to use bonemeal somewhere on the ground in order to grow plants as they would naturally if the player used bonemeal.
+
+The amount of planting positions on these fields are usually unlimited, this is because Mojang directly affects how much growth a bonemeal action can already do, although I wouldn't make the fields too big for no reason, if you make patches of at most 7x7 you should be fine.
+
+This one requires a bit more detail as every plant works slightly different.
+
+| Plant          | Work tag location                                                                                                                                                                       |
+|----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Seagrass       | For seagrass, the block below the water line itself should be tagged, the planter is supposed to bonemeal the dirt block itself.                                                        |
+| Sea pickles    | For sea pickles, the block below the water line itself should be tagged, however the planter will initially place the pickles, and then bonemeal the pickles directly to let them grow. |
```

Comment:
These could benefit from an image... is "the block below the water line" the water block itself, or the ground block underwater?